### PR TITLE
[10][IMP] purchase_request_to_rfq (add currency and payment term)

### PR DESCRIPTION
**Pull request by hparfr**
Add currency_id, payment_term and improve fiscal position when creating a purchase order

### CODE REVIEW

File: purchase_request_to_rfq/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "author": "Eficent, "
               "Acsone SA/NV,"
               "Odoo Community Association (OCA)",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.0.1",
     "website": "http://www.eficent.com",
```
Comment (nikul-serpentcs) [inline purchase_request_to_rfq/__manifest__.py]: https://github.com/OCA/purchase-workflow

**Comment by hparfr** [comment]
fixes #524
@lreficent

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review

**Review by PierrickBrun — APPROVED** [review]

File: purchase_request_to_rfq/__manifest__.py (hunk 1 above)
Comment (hparfr) [reply purchase_request_to_rfq/__manifest__.py]: @nikul-serpentcs fixed

**Comment by hparfr** [comment]
ping @hveficent

**Review by HviorForgeFlow — APPROVED** [review]
/ocabot merge patch

**Comment by HviorForgeFlow** [comment]
@aheficent what do you think about port this change to v12?

@hparfr resolve conflicts. Thanks

**Comment by AaronHForgeFlow** [comment]
> @aheficent what do you think about port this change to v12?

Yes, good for me

**Comment by HviorForgeFlow** [comment]
@hparfr could your squash commits and add proper name-description?

**Comment by hparfr** [comment]
It's  due to github merging tool.

You wan me to rebase then force push  ?

**Comment by HviorForgeFlow** [comment]
@hparfr I want one commit to cherry-pick easily, yes, this is an option :)

**Comment by hparfr** [comment]
Interesting, when I do a git rebase --i it don't show merge commits.

**Pull request merged**